src/test/java/com/thangle/fakes/BookFakes.java

```diff
@@ -38,4 +51,17 @@ public static List<BookEntity> buildBookEntities() {
                 .mapToObj(_ignored -> buildBookEntity())
                 .toList();
     }
+
+    public static Book buildUpdatedBook(UUID id) {
+        return Book.builder()
+                .id(id)
+                .title(RandomStringUtils.randomAlphabetic(3, 10))
```

Comment:
```suggestion
                .title(randomAlphabetic(3, 10))
```

Dùng static import (và những chỗ khác)